src/test/java/test/dependent/issue949/Issue949InheritanceTest.java

```diff
@@ -0,0 +1,12 @@
+package test.dependent.issue949;
+
+import org.testng.annotations.Test;
+
+public class Issue949InheritanceTest extends InheritanceBaseTest {
```

Comment:
Could you move your new package into the inheritance one? 
Could you transform your test into an integration one (and then rename the *Test into *Sample)?